community-membership.md

```diff
@@ -64,19 +59,21 @@ remain active contributors to the community.
 - Have your sponsoring reviewers reply confirmation of sponsorship: `+1`
 - Once your sponsors have responded, your request will be reviewed by the [Kubernetes GitHub Admin team], in accordance with their [SLO]. Any missing information will be requested.
 
+\* _Excluding the [Contributor Playground repository]. It is configure to allow
```

Comment:
Thank you!
Is there a typo here around "it is configure..."?

Reply:
gah, thanks - updated